packages/eas-cli/src/uploads.ts

```diff
@@ -2,15 +2,10 @@ import assert from 'assert';
 import FormData from 'form-data';
 import fs from 'fs';
 import got, { Progress } from 'got';
-import md5File from 'md5-file';
```

Comment:
this package is not used anywhere else, you can remove it from package.json

Reply:
Done